docs/test/test.md

```diff
@@ -88,13 +88,22 @@ GOROOT=/usr/local/go
 GOPATH=/go
 ```
 
-Then copy or clone or link pouch source code to the GOPATH：
+Then copy or clone pouch source code to the GOPATH：
 
 ```
 # pwd
 /go/src/github.com/alibaba/pouch
 ```
 
+The following steps are also needed to make sure libnetwork package could be found:
+
+```
+BUILDPATH=/tmp/pouchbuild
```

Comment:
Please add the remainder: the system will clean all the file or dir in `/tmp` after restart.